tests/ro-crate-metadata.json

```diff
@@ -0,0 +1,467 @@
+                {
+                    "@id": "outputs/ERR034597_1.small.fq.trimmed.2P.fq"
+                },
+                {
+                    "@id": "outputs/ERR034597_1.small.fq.trimmed.2U.fq"
+                },
+                {
+                    "@id": "outputs/ERR034597_1.small_fastqc.html"
+                },
+                {
+                    "@id": "outputs/ERR034597_2.small_fastqc.html"
+                },
+                {
+                    "@id": "stdout.log"
+                },
+                {
+                    "@id": "stderr.log"
+                }
+            ],
+            "mainEntity": {
+                "@id": "https://raw.githubusercontent.com/sapporo-wes/yevis-cli/d81e0e38143c63ead17d475b85c9b639958b1b47/tests/CWL/wf/trimming_and_qc.cwl"
+            }
+        },
+        {
+            "@id": "ro-crate-metadata.json",
+            "@type": "CreativeWork",
+            "about": {
+                "@id": "./"
+            },
+            "conformsTo": [
+                {
+                    "@id": "https://w3id.org/ro/crate/1.1"
+                },
+                {
+                    "@id": "https://w3id.org/workflowhub/workflow-ro-crate/1.0"
+                }
+            ]
+        },
+        {
+            "@id": "https://raw.githubusercontent.com/sapporo-wes/yevis-cli/d81e0e38143c63ead17d475b85c9b639958b1b47/tests/CWL/wf/trimming_and_qc.cwl",
+            "@type": [
+                "File",
+                "SoftwareSourceCode",
+                "ComputationalWorkflow"
+            ],
+            "attachment": [
+                {
+                    "@id": "exe/fastqc.cwl"
+                },
+                {
+                    "@id": "exe/trimmomatic_pe.cwl"
+                }
+            ],
+            "description": {
+                "@id": "https://raw.githubusercontent.com/sapporo-wes/yevis-cli/d81e0e38143c63ead17d475b85c9b639958b1b47/README.md"
+            },
+            "input": [
+                {
+                    "@id": "#fastq_1"
+                },
+                {
+                    "@id": "#fastq_2"
+                }
+            ],
+            "name": "CWL_trimming_and_qc",
+            "output": [
+                {
+                    "@id": "#ERR034597_1.small.fq.trimmed.1P.fq"
+                },
+                {
+                    "@id": "#ERR034597_1.small.fq.trimmed.1U.fq"
+                },
+                {
+                    "@id": "#ERR034597_1.small.fq.trimmed.2P.fq"
+                },
+                {
+                    "@id": "#ERR034597_1.small.fq.trimmed.2U.fq"
+                },
+                {
+                    "@id": "#ERR034597_1.small_fastqc.html"
+                },
+                {
+                    "@id": "#ERR034597_2.small_fastqc.html"
+                },
+                {
+                    "@id": "stdout.log"
+                },
+                {
+                    "@id": "stderr.log"
```

Comment:
"stdout.log" and "stderr.log" are not formal parameters, so they need to be removed from here